cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -3437,8 +3489,13 @@ class TranslatedVarArg extends TranslatedNonConstantExpr {
 
   final override Instruction getInstructionSuccessorInternal(InstructionTag tag, EdgeKind kind) {
     tag = VarArgsVAListLoadTag() and
-    kind instanceof GotoEdge and
-    result = this.getInstruction(VarArgsArgAddressTag())
+    (
+      kind instanceof GotoEdge and result = this.getInstruction(VarArgsArgAddressTag())
+      or
+      // All load instructions could throw an SEH exception
+      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge e), true) and
+      kind.(ExceptionEdge).isSEH()
```

Comment:
I don't think a call to `va_arg` can ever generate a SEH exception? Can it?

Reply:
removed